sound/soc/intel/boards/sof_board_helpers.h

```diff
@@ -47,6 +49,10 @@ struct sof_card_private {
 
 int sof_intel_board_card_late_probe(struct snd_soc_card *card);
 
+int sof_intel_board_set_dmic01_link(struct device *dev,
+				    struct snd_soc_dai_link *link, int be_id);
+int sof_intel_board_set_dmic16k_link(struct device *dev,
+				     struct snd_soc_dai_link *link, int be_id);
```

Comment:
Can we combine sof_intel_board_set_dmic16k_link and sof_intel_board_set_dmic01_link?

Reply:
Not sure what you mean by "combine" @bardliao ? Maybe we can use the same helper but the dailink name and dai name need to be different for backwards compatibility.

Reply:
> Not sure what you mean by "combine" @bardliao ? Maybe we can use the same helper but the dailink name and dai name need to be different for backwards compatibility.

I mean use the same helper, just like @brentlu does now.